### PR TITLE
Feature/read device identitification

### PR DESCRIPTION
**Pull request by eeerin**
I was able to start a python server and test some of this readDeviceIdenitifcation code, I did find an issue with my code. I should have an actual machine available soon.

### CODE REVIEW

**Comment by yaacov** [comment]
hi, weird, this branch has conflicts, can you take a look ?

**Comment by eeerin** [comment]
Fixed, I forgot to update my forked branch.

File: index.js [hunk 1]
```diff
@@ -395,7 +398,11 @@ ModbusRTU.prototype.open = function(callback) {
                         break;
                     case 43:
                         // read device identification
+<<<<<<< HEAD
```
Comment (yaacov) [inline index.js]: ? are you sure :heart:

Comment (eeerin) [reply index.js]: 😱

Comment (yaacov) [reply index.js]: :-)

Comment (eeerin) [reply index.js]: now I am slightly more. ❤️

**Comment by yaacov** [comment]
Thanks !
will merge when green :+1:

**Comment by eeerin** [comment]
@yaacov I was finally able to test this with an actual machine and it works! Please publish when you can 💚

**Comment by yaacov** [comment]
:+1: :birthday: :cat: 
https://github.com/yaacov/node-modbus-serial/releases/tag/v7.8.0

**Comment by yadidgarbo** [comment]
Hello yaacov, Hello eeerin,

I've been trying to get used to it, but without success.
Could you show me a sample call to read function 43.

I would like to read the device information from a PM5560 meter.
This can be queried via an IP address and a port.
The device is also able to return function 43.

Thanks!

**Comment by yaacov** [comment]
> Could you show me a sample call to read function 43.

@yadidgarbo hi,

I added an example here:
https://github.com/yaacov/node-modbus-serial/blob/master/examples/device-identification.js

have fun :cake:

**Pull request merged**